examples/multi_blade_row_example.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2023 ANSYS, Inc. All rights reserved
+"""
+.. _multi_blade_row_example:
+
+Multi blade-row meshing example
+-------------------------------
+This basic example shows how to set up a multi blade-row meshing instance and execute it in parallel
+
+"""
+
+#################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform the required imports. It is assumed that the ``ansys-turbogrid-core``
+# package has been installed.
+
+import os
+
+from ansys.turbogrid.core.multi_blade_row.multi_blade_row import MBR
+
+#################################################################################
+# Create and use an MBR instance
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create an MBR instance, set it up for a multi-blade row case and execute.
+
+if __name__ == "__main__":
+    # Set working directory:
+    working_directory = os.getcwd()
+
+    # Set location for the NDF file
```

Comment:
There should not be a hard-coded path here, use something like "." instead. If absolutely necessary to use a full path (to illustrate what is needed) use a generic one e.g. "C:\Users\<name>\MBR_example".